### ISSUE TITLE
implement and evaluate persisting layer 

### ISSUE THREAD

**Issue by 9j3** [9j3/chai#39]
**Is your feature request related to a problem? Please describe.**
We do not have the opportunity right now to do persistance of any data. However, the project grows and now the point has come to implement a simple DAL (Data-Access-Layer) with the corresponding Persisting Layer.

**Describe the solution you'd like**
I'll set up a docker-compose file with a database (PostgreSQL) and setting up [typeORM](https://typeorm.io) as the DAL.

**Describe alternatives you've considered**
In-Memory storage was the way go since now. It's not very maintainable.

**Comment by bariod** [comment]
Are you sure that we want to do this at this point in time?
Even in the Antrag we've said that we do not want to persist data.

**Comment by 9j3** [comment]
@bariod I agree with you, but at that time the starting position was also somewhat different and less clearly defined. It is a pain to persist the individual messages in memory. With the help of TypeORM, however, this is child's play. I see only one benefit in it and maybe our project sponsor Bruno will be happy that we add PostgreSQL ;) I've implemented it and I'll send you my work later, so you can have a look at it. Let me hear what you think about it...

**Comment by 9j3** [comment]
I am suspending this adjustment until further notice. We will complete the project without persisting for the time being.
